Makefile

```diff
@@ -43,6 +43,7 @@ test-all: test test-gentype
 
 reanalyze:
 	reanalyze.exe -set-exit-code -all-cmt _build/default/res_syntax -suppress res_syntax/testrunner
+	reanalyze.exe -set-exit-code -all-cmt _build/default/jscomp -suppress res_syntax/testrunner -exclude-paths jscomp/super_errors,jscomp/outcome_printer,jscomp/ounit_tests,jscomp/ml,jscomp/js_parser,jscomp/frontend,jscomp/ext,jscomp/depends,jscomp/core,jscomp/common,jscomp/cmij,jscomp/bsb_helper,jscomp/bsb
```

Comment:
Is the `-suppress res_syntax/testrunner` still needed here?

Reply:
We don't want to be obliged to put annotations for e.g. raising possible exceptions, in the test runner.